### ISSUE TITLE
Button to create References on Profiles is hidden

### ISSUE THREAD

**Issue by joatu** [joatuapp/joatu-v2#262]
There is currently no ability to leave a reference on people's profiles.

Perhaps it is related to #197 and #199 ?

**Comment by rsmithlal** [comment]
This should be resolved now. Please test, confirm, and close

**Comment by joatu** [comment]
I do not believe this is resolved. I still do not see any capacity to add a reference.

**Comment by joatu** [comment]
@rsmithlal
